lisa/sut_orchestrator/azure/features.py

```diff
@@ -80,3 +87,45 @@ def _is_supported(self) -> bool:
         if not isinstance(self._node.os, supported_distro):
             return False
         return True
+
+
+class Sriov(AzureFeatureMixin, features.Sriov):
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        super()._initialize(*args, **kwargs)
+        self._initialize_information(self._node)
+
+    def _switch_sriov(self, platform: Platform, enable: bool) -> None:
+        network_client = get_network_client(platform)
+        compute_client = get_compute_client(platform)
+        node = compute_client.virtual_machines.get(
+            self._resource_group_name, self._node.name
+        )
+        for nic in node.network_profile.network_interfaces:
+            # get nic name from nic id
+            # '/subscriptions/[subid]/resourceGroups/[rgname]/providers/Microsoft.Network/networkInterfaces/[nicname]' # noqa: E501
```

Comment:
It's a reference, not a real link. So, it can be wrapped, and doesn't affect readability a lot.